controller/controller.js

```diff
@@ -12,4 +12,15 @@ exports.getTopics = (req, res) => {
     })
 }
 
+exports.getArticleById = (req, res, next) => {
+    const articleId = req.params;
+  
+    selectArticleById(articleId.article_id).then((result) => {
+        res.status(200).send({ article: result.rows });   
```

Comment:
Sending back result.rows[0] would be better here, as it will give us just the article object 👍